src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/common/ContactUsPageBase.java

```diff
@@ -4,6 +4,8 @@
 import org.openqa.selenium.WebDriver;
 
 public abstract class ContactUsPageBase extends AbstractPage {
+    
+    private String s;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "s" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1068)